cardano-node/cardano-node.cabal

```diff
@@ -149,8 +149,8 @@ library
                       , cardano-crypto-class
                       , cardano-crypto-wrapper
                       , cardano-git-rev ^>=0.2.2
-                      , cardano-ledger-alonzo
-                      , cardano-ledger-allegra
```

Comment:
I think the bounds can be removed now, because here are the versions chosen on a recent version of `master`:

```shell
→ ghc-pkg list | grep ledger-alonzo
    cardano-ledger-alonzo-1.8.0.0
→ ghc-pkg list | grep ledger-allegra
    cardano-ledger-allegra-1.4.1.0
```

Reply:
Agreed. I'll close this PR now.